src/formatDate.js

```diff
@@ -7,8 +7,56 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateElements = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
Consider destructuring `dateElements` directly into named variables for better readability, instead of assigning them to properties of `newFromDate` object. This will also eliminate the need for the `newFromDate` object.